src/scripts/advanced_overlay.user.js

```diff
@@ -55,12 +77,7 @@ if (window.top !== window.self) {
       img.style.opacity = oState.opacity / 100;
     };
 
-    const mainContainer = document
```

Comment:
Moved to https://github.com/PlaceDE-Official/place-overlay/pull/22/files#diff-af4920c0010114b300e83f3e59dd1eba073699a34397f27a0170570ea6ebd0a6R50-R55